packages/landing/src/config.map.ts

```diff
@@ -145,10 +145,14 @@ export class MapConfig extends Emitter<MapConfigEvents> {
     this.style = style ?? null;
     this.layerId = layerId.startsWith('im_') ? layerId.slice(3) : layerId;
     this.tileMatrix = tileMatrix;
-    if (labels == null) {
-      this.labels = layerId === 'aerial' && this.isDebug === false;
+    this.labels = false;
+
+    if (typeof labels === 'string') {
+      // enable labels for any string value other than "false"
+      if (labels !== 'false') this.labels = true;
     } else {
-      this.labels = labels !== 'false';
+      // if not in debug mode, show labels for the aerial layer by default
+      if (layerId === 'aerial' && !this.isDebug) this.labels = true;
```

Comment:
We generally try and avoid `!foo` as that triggers type cohersion causing you to think of all the horrible values that javascript would consider to be truthy.

We generally try to be as explicit as possible, so `this.isDebug === false` would be my preference